Documentation-Requirements.md

```diff
@@ -79,3 +80,21 @@ utility.
       ```
 
   ```
+
+# Images
+
+All binary images files must be in a standard and well supported format such
+as PNG. This format is preferred for vector graphics, such as diagrams, as it
+can store the information more efficiently leading to smaller file sizes. JPEG
+format images are acceptable but are more appropriate for storing photographic
+images.
+
+Where possible, images should be generated using freely available software.
+
+Every binary image file **MUST** be accompanied by the "source" file used to
+generate it. This guarantees it will be possible for an image to be modified
+by updating the source file and re-generating the binary format image file.
+
+Ideally the format of all image source files will be an open standard,
```

Comment:
Lines 98-100 suggested rewrite: 
Ideally, the format of all image source files is an open standard, non-binary one such as SVG. Text formats are highly preferable because you can manipulate and compare them with standard tools (e.g. `diff(1)`).